webapp/src/js/controllers/inbox.js

```diff
@@ -267,6 +267,7 @@ const moment = require('moment');
     ctrl.tours = [];
     ctrl.adminUrl = Location.adminPath;
     ctrl.setIsAdmin(Session.isAdmin());
+    ctrl.filteredModals = [];
```

Comment:
I don't think the variable name is specific enough now that it's out of context. What do you think about calling it `ctrl.modalsToShow` or something?